packages/decoder/lib/types.ts

```diff
@@ -267,6 +267,18 @@ export interface Transaction {
    * Data sent with the transaction, as hex string.
    */
   input: string;
+  /**
+   * A positive unsigned 8-bit number between 0 and 0x7f that represents the type of the transaction.
+   */
+  type?: number;
+  /**
+   * Defaulted to (2 * block.baseFeePerGas) + maxPriorityFeePerGas. The maximum fee per gas that the transaction is willing to pay in total.
```

Comment:
Aha, thanks for these comments!  I'd suggest removing the bits about the defaults, they don't make sense in the decoder context.

Reply:
That makes sense. thank you for catching it.